docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,285 @@
+
+| Name (Symbol)        	| Data Type and Possible Values                                                                                                                    	| Input Attribute 	| Output Attribute 	| Modified SecretBinding Search Labels                                                              	|
+|----------------------	|------------------------------------------------------------------------------------------------------------------------------------	|-----------------	|------------------	|---------------------------------------------------------------------------------------------------	|
+| Platform Region (PR) 	| string, platform regions as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment 	| true            	| false            	| hyperscalerType: `<providerType>_<PR>` or hyperscalerType: `<providerType>_<PR>_<CR>` if used with CR 	|
+| Cluster Region (CR)  	| string, cluster region as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment   	| true            	| false            	| hyperscalerType: `<providerType>_<CR>` or hyperscalerType: `<providerType>_<PR>_<CR>` if used with PR 	|
+| Eu Access (EU)       	| no value                                                                                                                           	| false           	| true             	| euAccess: true                                                                                    	|
+| Shared (S)           	| no value                                                                                                                           	| false           	| true             	| shared: true                                                                                      	|
+
+
+
+## Validation
+
+KEB validates HAP Rules during startup. If the configuration is invalid, KEB will not start and an error message will be displayed in the logs. 
+
+The constraints used for validation during KEB startup include:
+* Rules format check - all the rules must comply with the format specified above.
+* Every rule Entry must refer to an existing plan - if you define a rule with non existing pools then an error is returned if a during KEB startup
+* Every Rule Entry must refer to an existing pool.
+* Every supported plan needs at least one rule entry, if no rule entries for plan defined then error is returned for SKR.
+* Uniqnuess validation check - KEB checks that all rule entries are unique in the scope of the rule. 
+Specifying more than one rule with the same number of identification attributes is prohibited. 
+If more than one such attribute is specified then the error is returned during KEB startup. For more details see [Uniqnuess & Priority](#uniqnuess--priority) section. 
+
+
+An example of such failing configurations is presented below.
+
+// TODO:
```

Comment:
Provide examples: I left a ton of them last time

Reply:
Removed leftover todo. I have truncated examples to necessary minimum.